roboto/bot.py

```diff
@@ -735,6 +736,53 @@ async def stop_inline_message_live_location(
             ),
         )
 
+    async def send_venue(
+        self,
+        chat_id: Union[ChatID, str],
+        latitude: float,
+        longitude: float,
+        title: str,
+        address: str,
+        foursquare_id: Optional[str] = None,
+        foursquare_type: Optional[str] = None,
+        disable_notification: Optional[bool] = None,
+        reply_to_message_id: Optional[MessageID] = None,
+        reply_markup: Optional[ReplyMarkup] = None,
+    ) -> Message:
+        """sendVenue API method.
```

Comment:
I dont know if it is a problem, but maybe `sendVenue` on the docstring should be written in the same format as the function? (`send_venue`)

Reply:
The docstring reflects the version on the HTTP API.